src/rockstor/fs/btrfs.py

```diff
@@ -736,15 +842,34 @@ def qgroup_is_assigned(qid, pqid, mnt_pt):
     return False
 
 
+def share_pqgroup_assign(pqgroup, share):
+    """
+    Convenience wrapper to qgroup_assign() for use with a share object where
+    we wish to assign / reassign it's current db held qgroup to a passed
+    pqgroup.
+    :param pqgroup: pqgroup to use as parent.
+    :param share: share object
+    :return: qgroup_assign() result.
+    """
+    mnt_pt = '{}/{}'.format(settings.MNT_PT, share.pool.name)
+    return qgroup_assign(share.qgroup, pqgroup, mnt_pt)
+
+
 def qgroup_assign(qid, pqid, mnt_pt):
+    """
+    Wrapper for 'BTRFS, qgroup, assign, qid, pqid, mnt_pt'
+    :param qid: qgroup to assign as child of pqgroup
+    :param pqid: pqgroup to use as parent
+    :param mnt_pt: btrfs filesystem mountpoint (usually the associated pool)
+    """
     if (qgroup_is_assigned(qid, pqid, mnt_pt)):
         return True
 
     # since btrfs-progs 4.2, qgroup assign succeeds but throws a warning:
     # "WARNING: # quotas may be inconsistent, rescan needed" and returns with
     # exit code 1.
     try:
-        run_command([BTRFS, 'qgroup', 'assign', qid, pqid, mnt_pt])
+        run_command([BTRFS, 'qgroup', 'assign', qid, pqid, mnt_pt], log=True)
     except CommandException as e:
         wmsg = 'WARNING: quotas may be inconsistent, rescan needed'
         if (e.rc == 1 and e.err[0] == wmsg):
```

Comment:
same comment as the previous one.

Reply:
The marked code is on the addition of the log=True entry, I found this useful when working on the issue so left it in, especially given that I don't thing we are going to be done with quota rework for a bit.